opentelemetry-api/tests/test_app.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This test is in place to ensure the API is returning values that
+are valid. The same tests have been added to the SDK with
+different expected results. See issue for more details:
+https://github.com/open-telemetry/opentelemetry-python/issues/142
+"""
+import unittest
+
+from opentelemetry import metrics, trace
+
+
+class TestAppWithAPI(unittest.TestCase):
```

Comment:
Why is this called `TestAppWithAPI`? From that name I had expected it to run the example app (which would actually be nice too). I suggest the name `TestAPIOnlyImplementation`.

Reply:
Thanks for the suggestion, it's much clearer. I've renamed the files to test_implementation as well.